examples/lustre/test.bats

```diff
@@ -0,0 +1,76 @@
+load "${CHTEST_DIR}/common.bash"
+
+setup () {
+    scope full
+    prerequisites_ok lustre
+
+    if [[ $SLURM_JOB_ID ]]; then
+        if [[ -z $ch_lustre ]]; then 
+            pedantic_fail 'no lustre to bind mount'
+        fi
+    else
+        if [[ -z $ch_lustre ]]; then
+            skip 'no lustre to bind mount'
+        fi
+    fi
+
+    # Check lustre directory exists
+    if [[ ! -e $ch_lustre ]]; then
+        echo "${ch_lustre} does not exist"
+        exit 1
+    fi
+    # Check lustre directory is a directory
+    if [[ ! -d $ch_lustre ]]; then
+        echo "${ch_lustre} is not a directory"
+        exit 1
+    fi
+
+    binds=${ch_lustre}:/mnt/0
+    work_dir=/mnt/0/charliecloud_testing_directory
+    mkdir -p "${ch_lustre}/charliecloud_testing_directory"
```

Comment:
Also remove any existing files so you're starting with a clean slate. You're doing the right thing below, to avoid the dangerous `rm -Rf`. Move those commands into a function and call it both places.